web/src/components/molecules/Dashboard/Greeting.tsx

```diff
@@ -0,0 +1,31 @@
+import styled from "@emotion/styled";
+import React from "react";
+
+const Greeting: React.FC = () => {
+  return <DashboardCard>Welcome to Re:Earth CMS !</DashboardCard>;
```

Comment:
Are we not doing internationalization yet? I know it is setup, so a little curious why we are avoiding it at the moment.

Reply:
I worked on translations on a separate branch with the account page (where user can change their language preferences).
So after this PR will add that one